engine/src/main/java/io/camunda/zeebe/engine/processing/processinstance/ProcessInstanceModificationProcessor.java

```diff
@@ -45,13 +45,15 @@
     implements TypedRecordProcessor<ProcessInstanceModificationRecord> {
 
   private static final String ERROR_MESSAGE_PROCESS_INSTANCE_NOT_FOUND =
-      "Expected to modify process instance but no process instance found with key '%d'";
+      "Expected to modify instance of process but no process instance was found with key '%d'";
```

Comment:
🔧 `instance of process` but then no specific process is mentioned. Personally, I prefer how this message was defined before. 

I had previously mentioned alignment because I thought there was more we could align, like the unsupported part of the other message. But, after reading the other messages again I don't think we can align this message with the others.